tests/unit/Codeception/Lib/Driver/MysqlTest.php

```diff
@@ -51,6 +51,21 @@ public function tearDown()
         }
     }
 
+    /**
+     * If the method exists (PHPUnit 5) forward the call to the parent class, otherwise
+     * call `expectException` instead (PHPUnit 6)
+     */
+    public function setExpectedException($exception, $message = '', $code = null)
```

Comment:
`Codeception\Test\Unit` extends `\PHPUnit_Framework_TestCase`, so changing parent class should be straightforward. Please to update all tests if it doesn't cause any issues.